pkg/operator/controller.go

```diff
@@ -459,11 +458,6 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 			c.listers.ImagePrunerConfigs = informer.Lister()
 			return informer.Informer()
 		},
-		func() cache.SharedIndexInformer {
-			informer := kubeSystemKubeInformerFactory.Core().V1().ConfigMaps()
-			c.listers.InstallerConfigMaps = informer.Lister().ConfigMaps(kubeSystemNamespace)
```

Comment:
Can we get rid fo `InstallerConfigMaps` in the listers object?

Reply:
yes!

Reply:
done